resources/assets/less/bem/notification-popup-item.less

```diff
@@ -110,15 +117,16 @@
     }
 
     &--time {
+      pointer-events: initial;
```

Comment:
This should be just on the span, not the whole div (as it covers the whole width). Maybe with `u-hover` on the span instead.

(also it can still be a link if it's an `<a>` 👀 although that'll require additional styling